scripts/release/workflow/check-upgradeable.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if ! git log -1 --pretty=%B | grep -q "Transpile ${REFERENCE_COMMIT}"; then
+  echo "Expected 'Transpile ${REFERENCE_COMMIT}' but found '$(git log -1 --pretty=%B)'"
```

Comment:
Maybe instead of passing `REFERENCE_COMMIT` we do:

```suggestion
cd lib/openzeppelin-contracts
REFERENCE_COMMIT="$(git rev-parse --short HEAD)"
cd ../.../

if ! git log -1 --pretty=%B | grep -q "Transpile ${REFERENCE_COMMIT}"; then
  echo "Expected 'Transpile ${REFERENCE_COMMIT}' but found '$(git log -1 --pretty=%B)'"
```